model-optimizer/mo/back/offline_transformations.py

```diff
@@ -1,14 +1,27 @@
-#!/usr/bin/env python3
-
-# Copyright (C) 2018-2021 Intel Corporation
+# Copyright (C) 2021 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+import argparse
+
 if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    # parser.add_argument("--apply_low_latency", action="store_true")
+    # parser.add_argument("--apply_pruning", action="store_true")
+    parser.add_argument("--path_to_model")
```

Comment:
May be rename the command line to "--input_model" since we already use this name in the MO?

Reply:
Agree